frontend/src/views/Collection/index.tsx

```diff
@@ -65,7 +73,14 @@ const Collection: FC = () => {
     visibility,
   });
 
-  const { data: collection, isError, isFetching } = collectionState;
+  const { data, isError, isFetching } = collectionState;
+
+  const [deleteMutation] = useDeleteCollection(id, visibility);
+
+  if (data?.tombstone === true) {
+    router.push(ROUTES.HOMEPAGE + "?tombstoned_dataset_id=" + id);
```

Comment:
Ah since this is a side effect, we should do it in `useEffect()`, so we don't `router.push()` at every re-render

Reply:
Done!